packages/react-native/Libraries/NewAppScreen/components/Header.js

```diff
@@ -8,42 +8,67 @@
  * @format
  */
 
-import type {Node} from 'react';
+import type { Node } from "react";
```

Comment:
Please revert these formatting changle, single quotes is the internal style guide.